gov_uk_dashboards/components/plotly/card.py

```diff
@@ -5,7 +5,7 @@
 def card(children):
     """A rectangle with a grey background.
     Mostly used to wrap individual visualisations, e.g. a gauge."""
-    return html.Li(children, className="mini-card")
+    return html.Div(children, className="mini-card")
```

Comment:
Was there a list here at one point and it's been removed?

Reply:
No it was use with UL card row. Each row was a list